### ISSUE TITLE
Support for `-v, --var stringArray    set a variable (can be set more than once)` on `okteto destroy` and `okteto pipeline destroy`

### ISSUE THREAD

**Issue by mnevadom** [okteto/okteto#3112]
Right now we can send ENVs vars to the `okteto deploy` command with the `-v` option.

It could be interesting to be able to send those vars to the `okteto destroy` option

**Comment by pchico83** [comment]
@mnevadom do you have a use case in mind?

**Comment by mnevadom** [comment]
I don't know if it is very common, but in the `acceptance-testing` the destroy section is used to run a `job` from an IMAGE, which runs a `go command`. In this case to destroy given clusters names, but it could be a job with whatever.

https://github.com/okteto/acceptance-testing/blob/93bcc94c053cf3a26897c91b2a2772e003981906/integration/okteto-deploy.yml#L25

I adapted it to export the `image` to use in the job because we can get the ENV in the job like we do in the `deploy` section, and I added in the documentation a way to send the cluster names before running the `okteto destroy` command. 

`CLUSTER_NAMES="CL1, CL2" okteto destroy -f okteto-deploy.yml`

But maybe, if there are other cases like this, maybe this one is not a normal case, it would be fine to: 
- Use in the job the ENV: `${OKTETO_BUILD_INTEGRATION_IMAGE}`
- Run the `okteto deploy -v CLUSTER_NAME=CL1` to have access later in the `go command`
